src/robotlibcore.py

```diff
@@ -141,6 +131,29 @@ def get_keyword_documentation(self, name):
             doc = '{}\n\n{}'.format(doc, tags) if doc else tags
         return doc
 
+    def get_keyword_types(self, name):
+        kw = self.keywords[name]
+        if hasattr(kw, 'robot_types'):
+            robot_types = getattr(kw, 'robot_types')
```

Comment:
This could be just `kw.robot_types` after we've checked that the attribute exists with `hasattr`. Could possibly be inlined.